modules/javafx.base/src/test/java/test/javafx/collections/VetoableObservableListTest.java

```diff
@@ -89,109 +89,102 @@ protected void onProposedChange(List<String> added, int... removed) {
     }
 
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_add() {
-        list.add(null);
-
+        assertThrows(NullPointerException.class, () -> list.add(null));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_add_indexed() {
-        list.add( 1, null);
-
+        assertThrows(NullPointerException.class, () -> list.add(1, null));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_addAll_collection() {
-        list.addAll(Arrays.asList("a", null, "b"));
-
+        assertThrows(NullPointerException.class, () -> list.addAll(Arrays.asList("a", null, "b")));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_addAll() {
-        list.addAll("a", null, "b");
-
+        assertThrows(NullPointerException.class, () -> list.addAll(Arrays.asList("a", null, "b")));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_addAll_collection_indexed() {
-        list.addAll(1, Arrays.asList("a", null, "b"));
-
+        assertThrows(NullPointerException.class, () -> list.addAll(1, Arrays.asList("a", null, "b")));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_set() {
-        list.set(1, null);
-
+        assertThrows(NullPointerException.class, () -> list.set(1, null));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_setAll() {
-        list.setAll("a", null);
-
+        assertThrows(NullPointerException.class, () -> list.setAll("a", null));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_setAll_collection() {
-        list.setAll(Arrays.asList("a", null, "b"));
+        assertThrows(NullPointerException.class, () -> list.setAll(Arrays.asList("a", null, "b")));
     }
 
-    @Test(expected=NullPointerException.class)
-    @Ignore
+    @Test
+    @Disabled
     public void testNull_listIterator_add() {
-        list.listIterator().add(null);
-
+        assertThrows(NullPointerException.class, () -> list.listIterator().add(null));
     }
-    @Test(expected=NullPointerException.class)
-    @Ignore
+
+    @Test
+    @Disabled
     public void testNull_listIterator_set() {
         ListIterator<String> it = list.listIterator();
         it.next();
-        it.set(null);
+        assertThrows(NullPointerException.class, () -> it.set(null));
```

Comment:
even though this is not an equivalent change, I think it actually makes the test better.
good job!